commit ecb1fa93cf11c834703defb4b4113dd6b6f774ee
Author: Volker Krause <vkrause@kde.org>
Date:   Wed Nov 16 17:35:44 2022 +0100

    Forward compatibility with Qt 6
    
    - Remove the meta data entries related to ministro, we don't use that and
    it's gone in Qt 6.
    - Explicitly export the main activity, needed by newer Android SDKs.

diff --git a/src/app/android/AndroidManifest.xml b/src/app/android/AndroidManifest.xml
index b9a4eb35..4fb5254e 100644
--- a/src/app/android/AndroidManifest.xml
+++ b/src/app/android/AndroidManifest.xml
@@ -22,7 +22,8 @@
                   android:name="org.kde.itinerary.Activity"
                   android:label="@string/application_name_short"
                   android:windowSoftInputMode="adjustResize"
-                  android:launchMode="singleTop">
+                  android:launchMode="singleTop"
+                  android:exported="true">
 
             <intent-filter>
                 <action android:name="android.intent.action.MAIN"/>
@@ -100,7 +101,6 @@
             </intent-filter>
 
             <meta-data android:name="android.app.lib_name" android:value="itinerary-app"/>
-            <meta-data android:name="android.app.qt_sources_resource_id" android:resource="@array/qt_sources"/>
             <meta-data android:name="android.app.repository" android:value="default"/>
             <meta-data android:name="android.app.qt_libs_resource_id" android:resource="@array/qt_libs"/>
             <meta-data android:name="android.app.bundled_libs_resource_id" android:resource="@array/bundled_libs"/>
@@ -116,8 +116,6 @@
             <meta-data android:name="android.app.load_local_jars" android:value="-- %%INSERT_LOCAL_JARS%% --"/>
             <meta-data android:name="android.app.static_init_classes" android:value="-- %%INSERT_INIT_CLASSES%% --"/>
             <!--  Messages maps -->
-            <meta-data android:value="@string/ministro_not_found_msg" android:name="android.app.ministro_not_found_msg"/>
-            <meta-data android:value="@string/ministro_needed_msg" android:name="android.app.ministro_needed_msg"/>
             <meta-data android:value="@string/fatal_error_msg" android:name="android.app.fatal_error_msg"/>
             <meta-data android:value="@string/unsupported_android_version" android:name="android.app.unsupported_android_version"/>
 
